src/txorphanage.cpp

```diff
@@ -118,6 +139,8 @@ void TxOrphanage::EraseForPeer(NodeId peer)
         }
     }
     if (nErased > 0) LogPrint(BCLog::TXPACKAGES, "Erased %d orphan tx from peer=%d\n", nErased, peer);
+    if (!Assume(m_peer_bytes_used.count(peer) == 0)) m_peer_bytes_used.erase(peer);
```

Comment:
leave a comment that this is belt-and-suspenders only, the condition should fail since the item should have been already erased